app/screens/search/index.js

```diff
@@ -29,6 +30,9 @@ function makeMapStateToProps() {
         const {recent} = state.entities.search;
         const {searchPosts: searchRequest} = state.requests.search;
 
+        const serverVersion = state.entities.general.serverVersion;
+        const enableDateSuggestion = isMinimumServerVersion(serverVersion, 5, 2);
```

Comment:
This should be 5.3 actually

Reply:
it is set to 5.2 temporarily so we can test, when we have a 5.3 marked server version to run against we can update this value

Reply:
This should be set to 5.3 then before this PR is merged